ee/tabby-ui/app/auth/signup/components/admin-register-step.tsx

```diff
@@ -0,0 +1,23 @@
+import { cn } from '@/lib/utils'
+
+export default function AdminRegisterStep({
```

Comment:
nit: since this is only used in `admin-register`, it's ok to put it inside of that file.

Reply:
updated